toontown/spellbook/MagicWordIndex.py

```diff
@@ -286,7 +286,39 @@ def handleWord(self, invoker, avId, toon, *args):
         toon.b_setCogLevels([ToontownGlobals.MaxCogSuitLevel] * 4)
 
         return f"Successfully maxed {toon.getName()}!"
+    
+class RestockInventory(MagicWord):
+    aliases = ['allstuff', 'restockinv', 'maxinv', 'maxinventory', 'restock']
+    desc = 'Gives target all the inventory they can carry.'
+    execLocation = MagicWordConfig.EXEC_LOC_SERVER
+
+    def handleWord(self, invoker, avId, toon, *args):
+        toon.inventory.maxOutInv()
+        toon.d_setInventory(toon.inventory.makeNetString())
+        return ("Maxing out inventory for " + toon.getName() + ".")
+
+class EmptyInventory(MagicWord):
+    aliases = ['nostuff', 'emptyinv', 'zeroinv', 'zeroinventory']
+    desc = 'Gives target all the inventory they can carry.'
+    execLocation = MagicWordConfig.EXEC_LOC_SERVER
 
+    def handleWord(self, invoker, avId, toon, *args):
+        toon.inventory.zeroInv()
+        toon.d_setInventory(toon.inventory.makeNetString())
+        return ("Zeroing inventory for " + toon.getName() + ".")
```

Comment:
I recommend that you combine the two inventory classes into one class, It would make the inventory stuff slightly easier to maintain and not having to remember separate words. It can restock the entire inventory by default, while having the ability to empty out the inventory via an optional command argument.

Reply:
I''l work on that :)